rust/src/action/parquet_read/mod.rs

```diff
@@ -586,6 +595,7 @@ impl Action {
             "remove" => Action::remove(Remove::from_parquet_record(col_data)?),
             "txn" => Action::txn(Txn::from_parquet_record(col_data)?),
             "protocol" => Action::protocol(Protocol::from_parquet_record(col_data)?),
+            "cds" => Action::cdc(AddCDCFile::from_parquet_record(col_data)?),
```

Comment:
Should this be `"cdc"`?

```suggestion
            "cdc" => Action::cdc(AddCDCFile::from_parquet_record(col_data)?),
```

Reply:
fixed